src/tests/ftest/control/dmg_system_reformat.py

```diff
@@ -74,16 +79,16 @@ def test_dmg_system_reformat(self):
 
         # Check that engine starts up again
         self.log.info("<SERVER> Waiting for the engines to start")
-        self.server_managers[-1].detect_engine_start(host_qty=2)
+        self.server_managers[-1].detect_engine_start(hosts_qty=2)
```

Comment:
Suggestion: I think we could just use the default of "all hosts" to avoid hardcoding "2", right?
```suggestion
        self.server_managers[-1].detect_engine_start()
```